apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -26,43 +21,45 @@ export default function HomeScreen() {
 
   useWarmCache(account?.enclaveKeyName);
 
-  const nav = useNav();
-
-  const onScroll = (event: NativeSyntheticEvent<NativeScrollEvent>) => {
-    // event.
-    const { contentOffset } = event.nativeEvent;
-    if (contentOffset.y > 32) {
-      // Show full-screen history
-      nav.navigate("History");
-    }
-  };
-
-  const scrollViewRef = useRef<ScrollView>(null);
-  const onScrollEnd = () => {
-    if (scrollViewRef.current == null) return;
-    scrollViewRef.current.scrollTo({ y: 0, animated: true });
-  };
-
   if (account == null) return null;
 
   return (
-    <View style={ss.container.outerStretchScroll}>
-      <Header />
-
-      <AmountAndButtons account={account} />
+    <View style={ss.container.homeContainer}>
+      <View
+        style={{
+          paddingLeft: 50,
+          paddingRight: 50,
+        }}
```

Comment:
recommend pulling this out into a `StyleSheet`, see other components

Reply:
done